Modul doesnt load

matze6486 opened the pull request:
Hey,
i installed your module (clone + npm install) and i use the sample config with my own api and my own lat/long, but the module displays "load" the whole time.
her is my config. maybe you can fix my problem

{
			module: "MMM-OpenWeatherMapForecast",
			header: "Weather",
			position: "top_right",
			classes: "default everyone",
			disabled: false,
			config: {
			    apikey: "xxxxxxxxxxxxx",
			    latitude: "51.78",
			    longitude: "6.12",      
			    iconset: "4c",
			    concise: false,
			    forecastLayout: "table"
			  }
		},

MarcLandis committed:
readd precipitation percentage

MarcLandis committed:
add feels like temperature, option to include current day to forcast, bug fixes

MarcLandis committed:
fix snow precipitation icons

MarcLandis committed:
rename module and update dependencies

- rename module to MMM-OpenWeatherMapForecast
- remove deprecated request dependency and use needle instead
- remove npm dependency

MarcLandis committed:
update readme

MarcLandis committed:
add screenshots

MarcLandis committed:
fix readme

albertusQuelle commented:
For me its also not working !

sdetweil commented:
looks like the version check is 2.2.0, but we are only at 2.1.7

think the check should be 2.1.0 (when templating was introduced)

sisimomo committed:
Adding new icons sets: "6fa" - Icon set 6 Filled Animated    -    "6oa" - Icon set 6 Outlined Animated

Marking Dark Sky's animated icon set as legacy only, so options "useAnimatedIcons" and "animateMainIconOnly". You should now use the new animated icons sets.

Adding a new config option "mainIconset". To be used instead of the "animateMainIconOnly" option. Allow you to set icon set for only the current weather icon.

Correcting a small class error in file MMM-OpenWeatherMapForecast.njk.

Changing the iconsets.img image to an animated gif to be able to see new animated icons in the README.md.

Big thanks to basmilius who have made available all theses animated icons for free. To see all the available icons, go see is github page https://github.com/basmilius/weather-icons.

MarcLandis committed:
Merge pull request #1 from sisimomo/animated_weather_icons

Adding new icons sets: "6fa" - Icon set 6 Filled Animated, "6oa" - Icon set 6 Outlined Animated

Commit:
add endpoint config option; the v3.0 api requires a subscription with payment details, whereas v2.5 does not. to use v3, add "endpoint": "https://api.openweathermap.org/data/3.0/onecall" to module's config in config.js

dathbe committed:
Update imperial rain/snow accumulation from mm to in

changed the units and value of precipitation accumulation from millimeters to inches if units are imperial.

dathbe committed:
Update node_helper.js

Add ```WEATHER_UPDATED``` notification in an attempt to make this work with MMM-RAIN-MAP module

dathbe committed:
Update node_helper.js

Undo adding the WEATHER_UPDATED notification

dathbe committed:
Update package.json

Add keywords

dathbe committed:
Update package.json

indentation

dathbe committed:
Update MMM-OpenWeatherMapForecast.js

magic mirror > magicmirror2

dathbe committed:
Update README.md

magicmirror > magicmirror2; michmich > magicmirrororg

dathbe committed:
Update package.json

magicmirror > magicmirror2

dathbe committed:
Update README.md

add code type

dathbe committed:
Update README.md

add code language

MarcLandis committed:
Merge pull request #7 from dathbe/conform

Non-substantive changes per recommendations

dathbe committed:
Merge branch 'MarcLandis:master' into master

KristjanESPERANTO committed:
Fix syntax

MarcLandis committed:
Merge pull request #8 from KristjanESPERANTO/patch-1

Fix syntax

MarcLandis committed:
Merge pull request #6 from dathbe/master

Update imperial rain/snow accumulation from mm to in

sdetweil committed:
Update MMM-OpenWeatherMapForecast.js

change default. 2.5 being shut down end of May 24.

maybe no longer expose endpoint 

you can add issues section   to forks now.. please do.

dathbe committed:
Update MMM-OpenWeatherMapForecast.js

remove mainiconset from defaults

MarcLandis committed:
Merge pull request #9 from sdetweil/master

change default url

MarcLandis committed:
Merge pull request #10 from dathbe/defaults

Update defaults to eliminate error

MarcLandis committed:
Update README.md

MarcLandis committed:
Fix error handling

found by pure coincidence at 
https://forum.magicmirror.builders/topic/18956/module-mmm-openweathermapforecast-is-not-running

thx to @sdetweil

sdetweil commented:
well, this fix is incomplete, I only posted the quick  for user approach..

error COULD be not nullOR error could be null and response.statusCode COULD be NOT 200
should separate the tests
```js
if(!error){
  if(response.statusCode == 200){ // good
  }
  else (api error)
}else // some request processing error  
```

MarcLandis committed:
Create dependabot.yml

dependabot[bot] committed:
Bump moment from 2.29.1 to 2.30.1

Bumps [moment](https://github.com/moment/moment) from 2.29.1 to 2.30.1.
- [Changelog](https://github.com/moment/moment/blob/develop/CHANGELOG.md)
- [Commits](https://github.com/moment/moment/compare/2.29.1...2.30.1)

---
updated-dependencies:
- dependency-name: moment
  dependency-type: direct:production
  update-type: version-update:semver-minor
...

Signed-off-by: dependabot[bot] <support@github.com>

MarcLandis committed:
Merge pull request #11 from MarcLandis/dependabot/npm_and_yarn/moment-2.30.1

Bump moment from 2.29.1 to 2.30.1

dependabot[bot] committed:
Bump needle from 2.5.2 to 3.3.1

Bumps [needle](https://github.com/tomas/needle) from 2.5.2 to 3.3.1.
- [Release notes](https://github.com/tomas/needle/releases)
- [Commits](https://github.com/tomas/needle/commits)

---
updated-dependencies:
- dependency-name: needle
  dependency-type: direct:production
  update-type: version-update:semver-major
...

Signed-off-by: dependabot[bot] <support@github.com>

MarcLandis committed:
Merge pull request #12 from MarcLandis/dependabot/npm_and_yarn/needle-3.3.1

Bump needle from 2.5.2 to 3.3.1

MarcLandis committed:
update package-lock.json

KristjanESPERANTO committed:
Magic Mirror -> MagicMirror

KristjanESPERANTO committed:
Update URL

KristjanESPERANTO committed:
console -> Log

KristjanESPERANTO committed:
Add missing LICENSE file
It's MIT according to the package.json.

KristjanESPERANTO committed:
Remove needle, Remove outdated comments, Format

MarcLandis committed:
update README and add LICENSE

MarcLandis committed:
Merge branch 'master' into master

MarcLandis committed:
Merge pull request #13 from KristjanESPERANTO/master

Review

MarcLandis committed:
Delete LICENSE

MarcLandis committed:
Update dependabot.yml
